[2024-01-02T20:26:53.474+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T20:26:35.471342+00:00 [queued]>
[2024-01-02T20:26:53.488+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T20:26:35.471342+00:00 [queued]>
[2024-01-02T20:26:53.489+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:26:53.970+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 20:26:35.471342+00:00
[2024-01-02T20:26:53.980+0000] {standard_task_runner.py:57} INFO - Started process 5152 to run task
[2024-01-02T20:26:53.992+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T20:26:35.471342+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpdf25xjb9']
[2024-01-02T20:26:54.085+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask transform_data
[2024-01-02T20:26:54.271+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T20:26:35.471342+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:26:56.695+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T20:26:35.471342+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T20:26:35.471342+00:00'
[2024-01-02T20:26:56.747+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:26:56.762+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 53, in transform_data
    data = data.drop('address', 1)
TypeError: drop() takes from 1 to 2 positional arguments but 3 were given
[2024-01-02T20:26:56.780+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T202635, start_date=20240102T202653, end_date=20240102T202656
[2024-01-02T20:26:56.871+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 22 for task transform_data (drop() takes from 1 to 2 positional arguments but 3 were given; 5152)
[2024-01-02T20:26:56.952+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:26:57.005+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
